assets/css/_section.scss

```diff
@@ -0,0 +1,137 @@
+@import '_variables';
```

Comment:
no need for multiple import of variable and mixin , just import in style.scss one time